physics/docs/pdftxt/srw3_phy_update.txt

```diff
@@ -0,0 +1,112 @@
+/**
+\page ccpp_phy Physics Update Summary
+
+This page provides an overview of the major updates in CCPP Physics since UFS-SRW v2.2.0.
+
+\b General \b Changes:
+- New supported suites:  RRFS_sas and HRRR_gf suites 
+- \ref CLM_LAKE_page in HRRR, HRRR_gf and RRFS_sas suites
+- New \ref SMK_DUST in HRRR, HRRR_gf and RRFS_sas suites
+- New photochem interface, which replace ozphys_2015 and h2ophys
+
+Suite RRFS_v1beta, which were supported with UFS-SRW v2.2.0, have been replaced with RRFS_sas.
+
+
+\b GFS  \b Scale-aware \b SAS  \b Cumulus \b Schemes:
+
+The updates between GFSv16 and GFSv17 are carefully outlined in Bengtsson and Han (2024) \cite Bengtsson_2024. The main updates include:
```

Comment:
Even though this wording is correct, I suggest this change to avoid user confusion with the GFS_v16 suite.

```suggestion
The updates since the previous release are carefully outlined in Bengtsson and Han (2024) \cite Bengtsson_2024. The main updates include:
```